modules/snooker/src/test/ts/atomic/operate/TransformOperationsTest.ts

```diff
@@ -10,12 +10,12 @@ import * as MockStructs from 'ephox/snooker/test/MockStructs';
 import TestGenerator from 'ephox/snooker/test/TestGenerator';
 
 UnitTest.test('TransformOperationsTest', () => {
-  let originalElements: Structs.ElementNew[] = [];
-  let expectedElements: Structs.ElementNew[] = [];
+  const originalElements: Structs.ElementNew[] = [];
+  const expectedElements: Structs.ElementNew[] = [];
 
   const clearElements = () => {
-    originalElements = [];
-    expectedElements = [];
+    originalElements.length = 0;
```

Comment:
This was a bug in this test, as it would have never passed through the new arrays to the `en` function and would have only passed the original reference. So we just set the length to 0 to clear the existing arrays instead.